packer

```diff
@@ -452,6 +452,14 @@ nap() {
   done
 }
 
+# Load default arguments from ~/.packerrc
+conffile=$HOME/.packerrc
+if [ -f $conffile ]; then
+	while IFS=\= read -r opt; do
+		export $opt=0
+	done < $conffile
```

Comment:
Just in case -- my two cents into this:
https://github.com/azat-archive/packer/commit/f7edb2d18c61a83c433d28e20a3b7c585d478db3